frontend/lib/js/category-trees/CategoryTreeNodeTextContainer.js

```diff
@@ -50,17 +54,30 @@ const CategoryTreeNodeTextContainer = (props: PropsType) => {
           <i className={classnames(
             'category-tree-node__icon',
             'fa', {
-              'fa-folder-open': !!props.open,
-              'fa-folder': !props.open
+              'fa-folder-open': !!props.open || searching,
+              'fa-folder': !props.open || !searching
```

Comment:
Consider the case `props.open == false` and `searching == true`. This will result in both classes being applied. So we need `!props.open && !searching`